requirements.txt

```diff
@@ -0,0 +1,2 @@
+lbt-ladybug==1.6.2
+ladybug-geometry==0.4.3
```

Comment:
I think you meant to put this as 1.6.2 but we don't need it as `lbt-ladybug` will take care of installing `ladybug-geometry`

Reply:
Ah so it does. But ladybug doesn't specify dependency version either: https://github.com/ladybug-tools/ladybug/blob/f4a7afea631f1f6e6e786d12dd4e8c5429d18405/setup.py#L19-L21

Reply:
I just merged https://github.com/ladybug-tools/ladybug/pull/155 which addresses the version specific dependency issue.

Reply:
Thanks!